docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,602 @@
+    register: kafka_acl_resp
+
+----
+
+. Configure the `create_kafka_topic` module to create a Kafka topic. The `kafka_id` field is a required field.
++
+.Example `create_kafka_topic` module
+[source,yaml]
+----
+- name: Create Kafka Topic
+    create_kafka_topic:
+      topic_name: "kafka-topic-name"
+      kafka_id: "{{ kafka_req_resp.id }}"
+      # To hardcode the kafka_id, uncomment and use the following line:
+      # kafka_id: "KAFKA_ID"
+      partitions: 1
+      retention_period_ms: "86400000"
+      retention_size_bytes: "1073741824"
+      cleanup_policy: "compact"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      create_topic_res_obj
+----
++
+. Update the configuration of the first topic using the `update_kafka_topic` module. In the following example, the cleanup policy has been updated from compact to delete by replacing `"compact"` with `"delete"` in the `cleanup_policy` field. You can also update the `retention_period_ms` and `retention_size_bytes` fields.
++
+.Example `update_kafka_topic` module
+[source,yaml]
+----
+- name: Update Kafka Topic
+    update_kafka_topic:
+      topic_name: "kafka-topic-name"
+      kafka_id: "{{ kafka_req_resp.id }}"
+      # To hardcode the kafka_id, uncomment and use the following line:
+      # kafka_id: "KAFKA_ID"
+      partitions: 1
+      retention_period_ms: "86400000"
+      retention_size_bytes: "1073741824"
+      cleanup_policy: "delete"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      update_topic_res_obj
+----
++
+
+. Configure the `delete_kafka_topic` module to delete both created topics.
++
+.Example `delete_kafka_topic` module
+[source,yaml]
+----
+- name: Delete Kafka Topic
+   rhoas.rhoas.delete_kafka_topic:
+     topic_name: "KAFKA_TOPIC_NAME"
+      kafka_id: "{{ kafka_req_resp_obj['kafka_id'] }}"
+      # To hardcode the kafka_id, uncomment and use the following line:
+      # kafka_id: "KAFKA_ID"
+     openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+----
+
+. Configure the `delete_service_account_by_id` module to delete the service account.
++
+.Example `delete_service_account_by_id` module
+[source,yaml]
+----
+- name: Delete Service Account
+   rhoas.rhoas.delete_service_account_by_id:
+   # service_account_id: "service_account_id"
+  service_account_id: "{{ srvce_acc_resp_obj['client_id'] }}"
+
+  # openshift_offline_token: "OFFLINE_TOKEN"
+----
+
+. Deprovision and delete the {product-kafka} instance using the `delete_kafka_by_id` module.
++
+.Example `delete_kafka_by_id` module
+[source]
+----
+- name: Delete kafka instance by ID
+    rhoas.rhoas.delete_kafka_by_id:
+     kafka_id: "{{ kafka_req_resp_obj['kafka_id'] }}"
+     openshift_offline_token: "offline_token"
+----
+. Save your changes.
+. Open a terminal and enter the following command to run the example `rhoas_test` playbook:
++
+[source, shell]
+----
+$ ansible-playbook rhoas_test.yml
+----
+
```

Comment:
I think it's worth adding a sentence or two here to tell users what they see happen when they run the example playbook.